src/cloudant/design_document.py

```diff
@@ -49,6 +49,46 @@ def __init__(self, database, document_id=None):
             self.setdefault(prop, dict())
 
     @property
+    def st_indexes(self):
+        """
+        Provides an accessor property to the Cloudant geospatial
```

Comment:
Slightly picky, but don't we refer to `Cloudant Geospatial` with the capital G on Geospatial?